torch/_inductor/tiling_utils.py

```diff
@@ -144,6 +144,38 @@ def indexing_div_rep(
     return None
 
 
+def find_broadcast_var(
+    index: sympy.Expr, var_ranges: dict[sympy.Expr, int]
+) -> Optional[sympy.Expr]:
+    """
+    Try to find the variable that this index is broadcast over.
+    A broadcast pattern is one where consecutive values of a variable
+    access the same memory location (e.g., x // 10).
```

Comment:
In general `x % 10` can also be a broadcast?

`x %10` v.s. `x // 10` just picks different dimension

Reply:
In this case, x % 10 will be read as coalesced so should still work the same

Reply:
That won't be considered coalesced. see https://github.com/pytorch/pytorch/blob/cfe04255dee6974f37ed15a79dd96e1c3ffe3fb1/torch/_inductor/tiling_utils.py#L179